### PR TITLE
Feature/mcp 1962 disable endpoints

### PR DESCRIPTION
**Pull request by au2**
# Removes two end points from qa+ environments

### What was the problem?

Unused endpoints were going to be scanned in qa environments. These end points are useful in development so they are left  for dev and default environments.

### How does this fix it?

Endpoints disables using Profiles. Disabled end points are
1) health-data-access
2) fetch-claims

Also cleaned up descriptions since this open api doc will be sent to scanners. 

### Jira Tickets

- [MCP-1962](https://amida.atlassian.net/browse/MCP-1962)

## How to test this PR

- Verify end points are working as before in dev environment
- Set ENV = qa ad build
- Verify that you can not curl to the disabled end points.

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
- [ ] If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) and
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
- [ ] If changing software behavior, post a link to the PR in Slack channel #benefits-virtual-regional-office

### CODE REVIEW

**Review by dimitri-amida — DISMISSED** [review]
sorry, checked the wrong branch. Starting over

File: controller/src/main/java/gov/va/vro/controller/VroController.java [hunk 1]
```diff
@@ -36,8 +34,6 @@
 import java.io.InputStream;
 import java.util.Base64;
 import java.util.Date;
-import java.util.List;
-import java.util.stream.Collectors;
 
 @Slf4j
 @RestController
```
Comment (dimitri-amida) [inline controller/src/main/java/gov/va/vro/controller/VroController.java:39]: This controller now has some unused dependencies such as fetchClaimsService

File: controller/src/main/java/gov/va/vro/controller/DevController.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+import java.util.stream.Collectors;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@Profile("!qa & !sandbox & !prod")
+public class DevController implements DevResource {
+
+  private final CamelEntrance camelEntrance;
+  private final GeneratePdfRequestMapper generatePdfRequestMapper;
```
Comment (dimitri-amida) [inline controller/src/main/java/gov/va/vro/controller/DevController.java:34]: This is not used

File: controller/src/main/java/gov/va/vro/controller/DevController.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  private final PostClaimRequestMapper postClaimRequestMapper;
+
+  private final FetchClaimsService fetchClaimsService;
+
+  private final ClaimMetricsService claimMetricsService;
```
Comment (dimitri-amida) [inline controller/src/main/java/gov/va/vro/controller/DevController.java:39]: This is not used

**Review by dimitri-amida — APPROVED** [review]
LGTM - I spotted some unused dependencies, but the APIs run as expected in dev mode

File: controller/src/main/java/gov/va/vro/controller/DevController.java (hunk 2 above)
Comment (au2) [reply controller/src/main/java/gov/va/vro/controller/DevController.java]: removed

File: controller/src/main/java/gov/va/vro/controller/DevController.java (hunk 3 above)
Comment (au2) [reply controller/src/main/java/gov/va/vro/controller/DevController.java]: removed

File: controller/src/main/java/gov/va/vro/controller/VroController.java [hunk 4]
```diff
@@ -9,11 +9,9 @@
 import gov.va.vro.api.requests.HealthDataAssessmentRequest;
 import gov.va.vro.api.resources.VroResource;
 import gov.va.vro.api.responses.ClaimMetricsResponse;
```
Comment (wlin2019) [inline controller/src/main/java/gov/va/vro/controller/VroController.java:11]: If you just want to disable some endpoints in some environment, should you just use a feature flag to disable/enable them? This looks too complicated if the task is as the Jira ticket MCP-1962 says.

**Review by wlin2019 — APPROVED** [review]
I have trouble to test it in my local environment at present. 

I feel the approach for solving the problem stated in the Jira ticket is too complicated. There is an issue listed in this repo for using feature flag. I think that should be the approach to enable/expose endpoints in different environments. 

If there is an urgent need to get build process going, I have no objection to go for the moment. But if I read the Jira ticket, MCP-1962, correctly, I don't think this is the right approach.

**Pull request merged**